bravado/mapping/http_future.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+# Copyright (c) 2013, Digium, Inc.
+# Copyright (c) 2014, Yelp, Inc.
+#
+
+DEFAULT_TIMEOUT_S = 5.0
+
+
+class HttpFuture(object):
```

Comment:
This was to keep the interfaces easy to access: `http_client` -> `requests_client` & `fido_client` and `http_future` -> `requests_future` & `fido_future`.